apps/condo/domains/common/components/TelephonyContext.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useCallback, useEffect } from 'react'
+
+import { usePostMessageContext } from './PostMessageProvider'
+import { RequestHandler } from './PostMessageProvider/types'
+
+export const TelephonyContext = React.createContext({})
+
+export const TelephonyContextProvider = ({ children }) => {
+    const { addEventHandler } = usePostMessageContext()
+    
+    const handleMessage = useCallback<RequestHandler<'CondoWebAppCall'>>(async ({ phone }) => {
+        if (typeof window === 'undefined') return null
+
+        window.open(`/phone/+${phone}`, '_blank')
```

Comment:
If we are on another tab, then this new window will in focus?

Reply:
Yes